recipes-core/packagegroups/packagegroup-xenclient-dom0.bb

```diff
@@ -110,6 +110,9 @@ RDEPENDS_${PN} = " \
     udev-extraconf-dom0 \
     vglass \
     disman \
+    libseccomp \
```

Comment:
libseccomp should be added automatically as a varstored rdepends (there aren't any command line tools packaged inside that are needed, right?).  With proper RDEPENDS in place, this patch can go away.